### PR TITLE
Create getExperimentalTorsions() function

### PR DESCRIPTION
**Pull request by greglandrum**
Currently the only way to figure out which bonds are being matched to which experimental torsion is by turning on verbose mode and parsing the log. This is unsatisfactory

This PR adds an API function to return information about which bonds match experimental torsions and which torsion they match.

### CODE REVIEW

**Comment by greglandrum** [comment]
@sriniker this is the last of the current set of confgen changes. This one adds functionality to retrieve the list of experimental torsions (and their parameters) which are identified for a molecule.

File: Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp [hunk 1]
```diff
@@ -215,6 +216,40 @@ void setBoundsMatrix(DGeomHelpers::EmbedParameters *self,
       new DistGeom::BoundsMatrix(nrows, sdata));
 }
 
+python::tuple getExpTorsHelper(const RDKit::ROMol &mol,
+                               bool useExpTorsions = false,
```
Comment (bp-kelley) [inline Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp:220]: It seems odd that useExpTorsions defaults to false.

What does this mean in this context?  The python wrapper defaults to True I think.

Comment (greglandrum) [reply Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp:220]: > It seems odd that useExpTorsions defaults to false.
> 
> What does this mean in this context? The python wrapper defaults to True I think.

This is just the helper function which is called by the python wrappers. The default values here don't mean anything.
It is confusing though, so I will remove them.

**Review by sriniker — COMMENTED** [review]
Looks good, just one small suggestion

File: Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp [hunk 2]
```diff
@@ -215,6 +216,39 @@ void setBoundsMatrix(DGeomHelpers::EmbedParameters *self,
...
+    d["smarts"] = std::get<2>(pr)->smarts;
+    d["V"] = std::get<2>(pr)->V;
+    d["signs"] = std::get<2>(pr)->signs;
+    d["atomIndices"] = std::get<1>(pr);
+    result.append(d);
+  }
+  return python::tuple(result);
+}
+
+python::tuple getExpTorsHelper2(
```
[the remark this answers is not in the document]
Comment (greglandrum) [reply Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp:245]: It's not strictly necessary, but it makes declaring the python functions later easier than doing an overload.
I'll rename it to something more descriptive

**Review by sriniker — APPROVED** [review]

**Review by sriniker — APPROVED** [review]

**Pull request merged**